src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/types/index.ts

```diff
@@ -134,12 +134,17 @@ export interface IFormTextAreaComponent extends IFormComponent { }
 
 export interface ILayoutSettings {
   pages: IPagesSettings;
+  components: IComponents;
```

Comment:
Consider renaming to `IComponentsSettings `for consistency. IComponents sounds like a list of components.